experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -436,5 +436,40 @@
 			"src/feature-libraries/editable-tree/editableTree.ts",
 			"src/feature-libraries/untypedTree.ts"
 		]
+	},
+	{
```

Comment:
It's not ideal to be adding more cycles, but as far as I can tell this is currently just a symptom of our imports being generally disorganized across the tree package and the various API layers, rather than an actual illegal import. The import in question is `[typeNameSymbol]`, which ought to be a legal import here eventually.

Reply:
This factory project should enable us to delete/replace contextuallyTyped with it when its done, and thats the real source of the cycle (that it depends on editable tree 1 to implement logic we want to delete as part of the editable tree 2 migration). SO that stuff should be deleted twice :)

Reply:
Excellent! :) Always good to delete things twice to make sure they really stay deleted. Just like pressing "C" multiple times in a row on a calculator.